### PR TITLE
better error handling for missing SOAP elements

### PR DESCRIPTION
**Pull request by prasunsultania**
Better error handling to speed up debugging:
1. SOAP message missing envelope and body 
2. request/response tests incorrect SOAP method name

### CODE REVIEW

File: test/request-no-envelope-body-test.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var request = require('request');
+var assert = require('assert');
+var http = require('http');
+var soap = require('../');
+var server;
+var port;
+
+
```
Comment (jsdevel) [inline test/request-no-envelope-body-test.js]: remote this empty line.

File: test/request-no-envelope-body-test.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      }, wsdl);
+      done();
+    });
+  });
+
+  after(function(){
+    server.close();
+  });
+
+
```
Comment (jsdevel) [inline test/request-no-envelope-body-test.js]: remove this empty line

**Comment by prasunsultania** [comment]
@jsdevel have removed the empty line and squashed the commits into one.

**Comment by jsdevel** [comment]
Thanks @prasunsultania !  See my other comments.

**Comment by prasunsultania** [comment]
@jsdevel have made changes by removing all empty lines as suggested. PLMK if it looks good.

Strange that `npm run coveralls failed` during the build with some error related to coverall itself `Bad response: 422 {"message":"Couldn't find a repository matching this job.","error":true}`

Is their a way to re-run the build check, as the build failed only for node.js 4 and does't seems to be an error with code or test cases.

**Comment by jsdevel** [comment]
tests are failing.  Can you fix them?

**Comment by prasunsultania** [comment]
@jsdevel Please see the [logs](https://travis-ci.org/vpulim/node-soap/jobs/151709238) , tests are passing. It is the coveralls which is failing. Is there a way to re-run travis build ?

**Comment by jsdevel** [comment]
Ahh, I see.  I kicked off the failed job again.  Hopefully it passes.

**Comment by prasunsultania** [comment]
@jsdevel it did pass :)

**Comment by jsdevel** [comment]
\o/

**Comment by jsdevel** [comment]
Thanks a ton @prasunsultania !

**Pull request merged**